scrapyd_k8s.sample-k8s.conf

```diff
@@ -12,6 +12,19 @@ http_port    = 6800
 # Kubernetes works with a remote repository and the Kubernetes launcher.
 repository   = scrapyd_k8s.repository.Remote
 launcher     = scrapyd_k8s.launcher.K8s
+object_storage_provider  = scrapyd_k8s.object_storage.LibcloudObjectStorage
```

Comment:
It's strange that you include this line here, but not in the docker config file.
I think we can leave this out this line, as there is only one implementation, and there is a sensible default (great!)

I would say that the job logging feature is an additional option, let's disable it by default, and remove it from the example configfiles. We might add it fully commented out (near the end of the file), to show how it could be configured, but I'd rather put that in documentation for now.